.github/CHANGELOG.md

```diff
@@ -98,6 +98,9 @@
   method.
   [(#564)](https://github.com/XanaduAI/strawberryfields/pull/564)
 
+* Cleanup `backends/tfbackend/ops.py` todo items.
```

Comment:
I would make this more specific - the changelog is intended to give users and developers a quick overview of any changes since the last release, without them needing to look at the code :)

So better to instead list the specific changes/improvements made.